modules/installation-initializing.adoc

```diff
@@ -345,7 +346,7 @@ ifdef::openshift-origin[]
 This field is optional.
 endif::[]
 endif::rhv[]
-ifeval::["{context}" == "installing-gcp-user-infra"]
+ifdef::gcp,azure-user-infra[]
```

Comment:
On second thought, since a simple `gcp` ifdef here is gonna hit more than just the originally-intended `installing-gcp-user-infra` context, I'd suggest adding a new `gcp-user-infra` attribute like you did for Azure. So for example at the top of the module where it currently has:

```
ifeval::["{context}" == "installing-gcp-user-infra"]
:gcp:
endif::[]
```

Add in `gcp-user-infra`:

```
ifeval::["{context}" == "installing-gcp-user-infra"]
:gcp:
:gcp-user-infra:
endif::[]
```

Then down here in the "Optional" step, update the ifdef to look like:

```
ifdef::gcp-user-infra,azure-user-infra[]
```

That way the "Optional" step will continue only _only_ show up in that `installing-gcp-user-infra` module.

Reply:
Incorporated